sdk/python/feast/infra/feature_servers/multicloud/prebuild-power.sh

```
@@ -0,0 +1,178 @@
+python${PYTHON_VERSION} -m pip install -r requirements.txt
+export GRPC_PYTHON_BUILD_SYSTEM_OPENSSL=1
+python${PYTHON_VERSION} -m build --wheel --no-isolation
+cp dist/*.whl /wheelhouse/
+cd ..
+
+#######################################################
+# Build Pyarrow  (Python package)
+#######################################################
+echo "Building pyarrow..."
+dnf install -y https://mirror.stream.centos.org/9-stream/BaseOS/ppc64le/os/Packages/centos-gpg-keys-9.0-24.el9.noarch.rpm \
+https://mirror.stream.centos.org/9-stream/BaseOS/`arch`/os/Packages/centos-stream-repos-9.0-24.el9.noarch.rpm \
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)